.github/workflows/pre-main.yaml

```diff
@@ -483,9 +483,31 @@ jobs:
           username: ${{ secrets.QUAY_ROBOT_USERNAME }}
           password: ${{ secrets.QUAY_ROBOT_TOKEN }}
 
-      - name: (if on main and upstream) Push the newly built image to Quay.io
+      - name: Build the `cnf-certification-test` images for multi-platform (x86)
         if: ${{ github.ref == 'refs/heads/main' && github.repository_owner == 'test-network-function' }}
-        run: docker push --all-tags ${REGISTRY}/${TNF_IMAGE_NAME}
+        uses: nick-fields/retry@v3
+        with:
+          timeout_minutes: 90
+          max_attempts: 3
+          command: make build-image-local-x86
+        env:
+          IMAGE_TAG: ${TNF_IMAGE_TAG}
+
+      - name: Build the `cnf-certification-test` images for multi-platform (arm64)
+        if: ${{ github.ref == 'refs/heads/main' && github.repository_owner == 'test-network-function' }}
+        uses: nick-fields/retry@v3
+        with:
+          timeout_minutes: 90
+          max_attempts: 3
+          command: make build-image-local-arm
+        env:
+          IMAGE_TAG: ${TNF_IMAGE_TAG}
```

Comment:
You can combine two commands into one step.

Reply:
I thought about that but I figured I wanted the granularity to be able to see which step fails if there are any problems.